include/nbl/asset/IDescriptorSet.h

```diff
@@ -57,29 +59,34 @@ class NBL_API IDescriptorSet : public virtual core::IReferenceCounted
                 };
```

Comment:
remove the OpenGL comment about `imageLayout`

Reply:
Resolved.